src/main/java/bankservice/demo/repository/RoleRepository.java

```diff
@@ -0,0 +1,12 @@
+package bankservice.demo.repository;
+
+import bankservice.demo.model.Role;
+import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+
+public interface RoleRepository extends JpaRepository<Role, Long> {
+    @Query("select r from roles r where r.roleName = :name")
+    Optional<Role> getRoleByRoleName(@Param("name") String name);
```

Comment:
```suggestion
    Optional<Role> getByRoleName(Role.RoleName name);
```
Spring Data JPA is capable of automatically build the query in many cases based on the method name and parameters)